src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const nodeList = document.querySelectorAll('li[data-salary]');
+
+function createEmployeeArray(employees) {
```

Comment:
Follow the naming according to the task description
![image](https://github.com/mate-academy/js_task_parse_list_DOM/assets/96409563/b59950bd-7753-40e3-a514-9ced04a191b9)